process/factory/shard/vmContainerFactory_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build !race
```

Comment:
why do we have two go compiler directives here?

go:build !race and +build !race ?

Reply:
removed the compiler directives & also the TODO. Arwen is fixed